subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -792,19 +792,17 @@ static void response_cb(struct http_response *rsp,
 			hb_context.dl.http_content_size = rsp->content_length;
 		}
 
-		if (rsp->body_found == 1) {
-			if (body_data == NULL) {
-				body_data = rsp->recv_buf;
-				body_len = rsp->data_len;
-			}
-		}
+		if ((rsp->body_found == 1) && (body_data == NULL)) {
```

Comment:
If `body_data` is not NULL, then we do not call `flash_img_buffered_write()`, is this expected behaviour?

Reply:
during first response callback, data is missed. resolved now